simulators/nav/src/components/README.md

```diff
@@ -1,9 +1,9 @@
 # <img src="../static/mrover.png" alt="MRover Logo" width="30"/> MRover Navigation Simulator
 
 ## Table of Contents
-[Directory Overview](#directory-overview)  
```

Comment:
lines 4-6 seemingly mistakenly removed a line break character, so these three headers render on one line in the wiki

Reply:
Good catch! My text editor must have removed this trailing whitespace without me realizing. I also noticed this problem in another nav sim readme so I'll fix both.